tsconfig.json

```diff
@@ -13,6 +13,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
+    "baseUrl": ".",
```

Comment:
(Ah, maybe the relative paths just need to be updated)

Reply:
Will have this fixed in a sec